mdLinkChecker/mdLinkChecker.ts

```diff
@@ -0,0 +1,200 @@
+/**
+ * validate relative urls and external links in markdown files
+ *
+ */
+import validator = require("validator");
```

Comment:
why not use import * as validator from "validator"?

Reply:
Fixed.